### PR TITLE
admin/replace-heavy-nlp-deps-with-stemmer

### PR DESCRIPTION
**Pull request by evamaxfield**
**Pull request recommendations:**
- [x] Name your pull request _your-development-type/short-description_. Ex: _feature/read-tiff-files_
- [x] Link to any relevant issue in the PR description. Ex: _Resolves [gh-12], adds tiff file format support_
- [x] Provide context of changes.

The build "succeeded" but fails on the actual live website due to weird stupid large dependencies that we really don't need (I think)

- [ ] Provide relevant tests for your feature or bug fix.
- [ ] Provide or update documentation for any feature added by your pull request.

Thanks for contributing!

### CODE REVIEW

**Comment by evamaxfield** [comment]
Tested deployment here: https://jacksonmaxfield.github.io/seattle/#/

IT WORKS THIS TIME :tada:

File: package.json [hunk 1]
```diff
@@ -22,11 +21,10 @@
     "react-virtualized": "^9.21.1",
     "request": "^2.88.2",
     "semantic-ui-react": "^0.88.1",
+    "stemmer": "^1.0.5",
```
Comment (tohuynh) [inline package.json:24]: Is this stemmer compatible with cdptools' stemmer?

**Review by tohuynh — COMMENTED** [review]
about previous pr:
```
const Banner = () => (
  <Container style={{ backgroundColor: "#ededf0", padding: "1em 1em", width: "100vw"}}>
      <p>
          Support this project by&nbsp;
          <a
            href="https://forms.gle/eGyjhPx4hU3UBidKA"
            style={{ textDecoration: "underline" }}
          >
            giving your feedback &gt;
          </a>
      </p>
  </Container>
);
```

This is a little bit better to read?

**Comment by evamaxfield** [comment]
> about previous pr:
> 
> ```
> const Banner = () => (
>   <Container style={{ backgroundColor: "#ededf0", padding: "1em 1em", width: "100vw"}}>
>       <p>
>           Support this project by&nbsp;
>           <a
>             href="https://forms.gle/eGyjhPx4hU3UBidKA"
>             style={{ textDecoration: "underline" }}
>           >
>             giving your feedback &gt;
>           </a>
>       </p>
>   </Container>
> );
> ```
> 
> This is a little bit better to read?

I put the `&nbsp` to make sure that the link is always on a single line even on small screens. Which makes it imo look a lot better. Try it in responsive mode on various devices. Looks better with it all on one line than split imo.

File: package.json (hunk 1 above)
Comment (evamaxfield) [reply package.json:24]: It is! They both use the same underlying porter stemmer alg.

**Comment by evamaxfield** [comment]
But I will change the `>` to `&gt;`

**Comment by tohuynh** [comment]
> > about previous pr:
> > ```
> > const Banner = () => (
> >   <Container style={{ backgroundColor: "#ededf0", padding: "1em 1em", width: "100vw"}}>
> >       <p>
> >           Support this project by&nbsp;
> >           <a
> >             href="https://forms.gle/eGyjhPx4hU3UBidKA"
> >             style={{ textDecoration: "underline" }}
> >           >
> >             giving your feedback &gt;
> >           </a>
> >       </p>
> >   </Container>
> > );
> > ```
> > 
> > 
> > This is a little bit better to read?
> 
> I put the `&nbsp` to make sure that the link is always on a single line even on small screens. Which makes it imo look a lot better. Try it in responsive mode on various devices. Looks better with it all on one line than split imo.

I see. Yea it does look better. but can we change  `>` to `&gt;`. the `>` is giving an error message in my editor (no big deal)

**Review by tohuynh — APPROVED** [review]

**Pull request merged**